lib/fixup.rb

```diff
@@ -40,7 +40,10 @@
   { pkg_name: 'acli', pkg_rename: 'acquia_cli', pkg_deprecated: nil, comments: 'Renamed to better match upstream.' },
   { pkg_name: 'agrind', pkg_rename: 'angle_grinder', pkg_deprecated: nil, comments: 'Renamed to better match upstream.' },
   { pkg_name: 'apriconv', pkg_rename: 'apr_iconv', pkg_deprecated: nil, comments: 'Renamed to better match upstream.' },
-  { pkg_name: 'aprutil', pkg_rename: 'apr_util', pkg_deprecated: nil, comments: 'Renamed to better match upstream.' }
+  { pkg_name: 'aprutil', pkg_rename: 'apr_util', pkg_deprecated: nil, comments: 'Renamed to better match upstream.' },
+  { pkg_name: 'bz2', pkg_rename: 'bzip2', pkg_deprecated: nil, comments: "Renamed to better match upstream." },
+  { pkg_name: 'bz3', pkg_rename: 'bzip3', pkg_deprecated: nil, comments: "Renamed to better match upstream." },
+  { pkg_name: 'codium', pkg_rename: 'vscodium', pkg_deprecated: nil, comments: "Renamed to better match upstream." }
```

Comment:
Use single instead of double quotes?

Reply:
yeah, Rubocop should have noticed this! Why didn't it?

Do a `rubocop -c .rubocop.yml -A lib/fixup.rb` to clean this up real quick...